### PR TITLE
pythonPackages.blis: 0.2.4 -> 0.4.0, pythonPackages.thinc: 7.0.8 -> 7.1.0

### PR DESCRIPTION
**Pull request by danieldk**
###### Motivation for this change

Update the blis and thinc dependencies of spaCy. Tested using the `en_core_web_sm` model

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by danieldk** [comment]
@GrahamcOfBorg build python3Packages.spacy

**Comment by aborsu** [comment]
Looks good to me, good job spotting the license issue.

**Review by aborsu — APPROVED** [review]

**Comment by risicle** [comment]
`nix-review` (merge checkout mode) happy for me on macos 10.13.

**Review by risicle — APPROVED** [review]

File: pkgs/development/python-modules/spacy/default.nix [hunk 1]
```diff
@@ -34,7 +34,9 @@ buildPythonPackage rec {
 
   prePatch = ''
     substituteInPlace setup.py \
-      --replace "plac<1.0.0,>=0.9.6" "plac>=0.9.6"
+      --replace "plac<1.0.0,>=0.9.6" "plac>=0.9.6" \
+      --replace "thinc>=7.0.8,<7.1.0" "thinc>=7.0.8" \
```
Comment (jonringer) [inline pkgs/development/python-modules/spacy/default.nix]: kind of a small point, but if you want to get rid of version bounds, you can just name the package itself
```suggestion
      --replace "thinc>=7.0.8,<7.1.0" "thinc" \
```

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
diff LGTM
spacy seems to work

```
[6 built, 0.0 MiB DL]
https://github.com/NixOS/nixpkgs/pull/67716
6 package were build:
python27Packages.blis python27Packages.thinc python37Packages.blis python37Packages.spacy python37Packages.textacy python37Packages.thinc
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build python27Packages.blis python27Packages.thinc python37Packages.blis python37Packages.spacy python37Packages.textacy python37Packages.thinc

**Comment by jonringer** [comment]
builds locally on nixos, builds on ofborg with darwin. LGTM

**Pull request merged**